sass/compat/themes/buddyboss-theme/_woocommerce-form-fields.scss

```diff
@@ -0,0 +1,12 @@
+// ------------------------------------
+// WOOCOMMERCE FORM-ROW
+// ------------------------------------
+
+body#{ $_body-theme-selector }#{ $_body-page-selector } div.woocommerce form .form-row input[type=text] {
+	z-index: 3;
```

Comment:
Medium: use SASS variables and functions when available.

Use plugin functions for `z-index` values, unless it needs to be set to a theme specific value in which case we need to add a comment at the end of the line explaining why the value is not using the functions or variables available.

This also applies to colors, spacing and other styles for which variables and functions are available from our plugin.